lib/modules/platform/gitlab/index.ts

```diff
@@ -1042,9 +1043,7 @@ export async function setBranchStatus({
 
   try {
     // give gitlab some time to create pipelines for the sha
-    await setTimeout(
-      parseInteger(process.env.RENOVATE_X_GITLAB_BRANCH_STATUS_DELAY, 1000),
-    );
+    await setTimeout(GlobalConfig.get('gitlabBranchStatusDelay', 1000));
```

Comment:
pass this as an initPlatform() parameter and save to local config instead of reading from GlobalConfig.get()